json_schema/schema/schema.json

```diff
@@ -175,7 +175,7 @@
             ],
             "title": "Processor"
         },
-        "LogRecordProcessorArgs": {
+        "ProcessorArgs": {
```

Comment:
Should we have a general `ProcessorArgs` type, or separate the types for `LogRecordProcessor` and `SpanProcessor`?

Also, the properties here are specific to batch processor. Will need to rethink how to define this type to accommodate simple, batch, and extension processors.

Reply:
I think ProcessorArgs is ok until we have a reason to be more specific.

I suspect we'll want per type of processor arguments, much like we'll want the same for exporters. Though I guess that will only support the known processor/exporter. We should plan to provide documentation on how one would define and publish their own schemas if they wanted to support additional components